moto/s3/responses.py

```diff
@@ -2140,7 +2140,7 @@ def _notification_config_from_body(self) -> Dict[str, Any]:
                         n["Event"] = [n["Event"]]
 
                     for event in n["Event"]:
-                        if event not in S3NotificationEvent.events():
+                        if not S3NotificationEvent.is_event_valid(event):
```

Comment:
This logic is now duplicated into the `Notification`-constructor. That seems to a more logical place to validate this value - so I think we can remove the check here. Or am I missing something?

Reply:
That also sounds good to me :) Thanks